src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -61,6 +63,13 @@
             "com/mathworks/ci/MatlabBuilder/runMatlabTests.m";
     private static final String AUTOMATIC_OPTION = "RunTestsAutomaticallyOption";
 
+    // Input parameter names (Passed to runMatlabTests.m as name-value pair arguments)
+    private static final String pdfReportParamName     = "'PDFReport'";
```

Comment:
+1 to Abhishek's comment here about having this be a struct of some sort. However, the matlab command the user sees would be less readable if we were to put this in a struct.

Reply:
I can create a Java map, but the main question is how to pass the map to runMatlabTests.m script? I am not sure if a MATLAB function can accept a java map or not. I need to analyze further to see how best we can incorporate this. 
Meanwhile I feel we can keep the name-value pairs here so it would be more readable for users.